spec/controllers/api/v2/commodities_controller_spec.rb

```diff
@@ -93,10 +93,32 @@
     context 'when a filter for meursing_additional_code_id is passed' do
       subject(:do_response) { get :show, params: { id: commodity.code, filter: { meursing_additional_code_id: '000' } } }
 
-      after { Thread.current[:meursing_additional_code_id] = nil }
+      let(:commodity) do
+        create(
+          :commodity,
+          :with_indent,
+          :with_chapter,
+          :with_heading,
+          :with_description,
+          :with_meursing_measures,
+          :declarable,
+        )
+      end
+
+      before do
+        Thread.current[:meursing_additional_code_id] = 'foo'
+      end
+
+      it 'sets the value to nil when we are done making the request' do
```

Comment:
small wording improvement:

sets the value to nil when the request completes
or
sets the value to nil when the request  is completed